common/src/main/kotlin/DiscordTimestamp.kt

```diff
@@ -3,7 +3,7 @@ package dev.kord.common
 import kotlinx.datetime.Instant
 
 fun Instant.toMessageFormat(style: DiscordTimestampStyle = DiscordTimestampStyle.ShortDateTime) =
-    "<t:${toEpochMilliseconds()}:${style.style}>"
+    "<t:${toEpochMilliseconds() / 1000}:${style.style}>"
```

Comment:
better use epochSeconds instead?

Reply:
```suggestion
    "<t:$epochSeconds:${style.style}>"
```
now i can suggest thanks github